pkg/controller/installation/installation_controller.go

```diff
@@ -154,6 +155,7 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 				log.Info("Error updating Installation resource. Requeue and retry.")
 				return reconcile.Result{
 					Requeue: true,
+					RequeueAfter: time.Second * 10,
```

Comment:
If this is used every where should be a constant. Is this needed for test reasons?

Reply:
as above, it just has to have a value. It's unrelated to tests, just noticed it while poking around with @sedroche